R/type-list-of.R

```diff
@@ -212,6 +227,19 @@ vec_cast.vctrs_list_of.vctrs_list_of <- function(x, to, ...) {
   list_as_list_of(x, ptype = ptype)
 }
 
+#' @export
+vec_cast.list.vctrs_list_of <-function(x, to, ...) {
+  vec_data(x)
```

Comment:
I think you should use the same thing in the `as.list.vctrs_list_of()` method, for consistency.

---

Also, maybe pull in `rlang:::vec_unstructure()` because we really don't need to pay for the `vec_assert()` and `vec_proxy()` calls from `vec_data()` - we just want to strip all attributes except names.

Reply:
`vec_unstructure()` dispatches 4 times. If we're looking for efficiency, I'll just add `list_of_unstructure()`.